src/analyses/apron/apronPriv.apron.ml

```diff
@@ -1071,6 +1073,8 @@ struct
     | VarQuery.Global g -> vf (V.global g)
     | _ -> ()
 
+  let invariant_vars ask getg st = protected_vars ask
+
```

Comment:
Same argument here! Any variant of mutex-meet should never have invalid information in the local state!